runtime/src/main/resources/application.yml

```diff
@@ -17,12 +17,6 @@
 deployment:
   file: io/syndesis/dao/deployment.json
 
-resteasy:
```

Comment:
Is it right that the main motivation of this PR is to avoid classpath scanning and so save some startup time ?